frontend/components/disclaimer.js

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
 
+const flagUrl = require('uswds/dist/img/us_flag_small.png');
+
 const Disclaimer = () =>
   <div className="usa-disclaimer">
     <div className="usa-grid">
       <span className="usa-disclaimer-official">
         <img
           className="usa-flag_icon"
           alt="US flag signifying that this is a United States Federal Government website"
-          src="/images/uswds/us_flag_small.png"
+          src={flagUrl}
```

Comment:
Hrm, it's apparently something with mocha not an actual problem. Trying to figure out how to make mocha not fail on it...